test/parallel/test-runner-output.mjs

```diff
@@ -203,6 +203,16 @@ const tests = [
     name: 'test-runner/output/arbitrary-output.js',
     flags: ['--test-reporter=tap'],
   },
+  {
+    name: 'test-runner/output/non-tty-forced-color-output.js',
+    transform: specTransform,
+  },
+  {
+    name: 'test-runner/output/assertion-color-tty.mjs',
+    flags: ['--test', '--stack-trace-limit=0'],
```

Comment:
Hey @cjihrig, I had to add the `--stack-trace-limit=0` flag because modifying
https://github.com/nodejs/node/blob/4d6d7d644be4f10f90e5c9c66563736112fffbff/test/common/assertSnapshot.js#L7-L9
to match colored stack traces also seems a little painful.

Do you think it's acceptable?